frontend/src/metabase/visualizations/visualizations/PivotTable.jsx

```diff
@@ -178,6 +181,10 @@ export default class PivotTable extends Component {
     },
   };
 
+  setBodyRef = element => {
+    this.bodyRef = element;
```

Comment:
Could probably use a better name here. Suggestions welcome.

Reply:
I think "body" is descriptive enough in the context of this visualization.